airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/csv_parser.py

```diff
@@ -132,6 +133,19 @@ class CsvParser(FileTypeParser):
     _MAX_BYTES_PER_FILE_FOR_SCHEMA_INFERENCE = 1_000_000
 
     def __init__(self, csv_reader: Optional[_CsvReader] = None):
+        # Increase the maximum length of data that can be parsed in a single CSV field. The default is 128k, which is typically sufficient
+        # but given the use of Airbyte in loading a large variety of data it is best to allow for the maximum field size possible to avoid
+        # skipping data on load. https://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072
+        max_int = sys.maxsize
+
+        while True:
+            # decrease the max_int value by factor 10 as long as the OverflowError occurs.
+            try:
+                csv.field_size_limit(max_int)
+                break
+            except OverflowError:
+                maxInt = int(max_int/10)
```

Comment:
why divide the value by 10? Why not set it to `sys.maxint`?

Reply:
I added some detail to that in my response above to Natik